src/test/groovy/com/palantir/gradle/graal/GradleGraalPluginIntegrationSpec.groovy

```diff
@@ -36,11 +36,11 @@ class GradleGraalPluginIntegrationSpec extends IntegrationSpec {
         buildFile << '''
             apply plugin: 'java'
             apply plugin: 'com.palantir.graal'
-            
+
             graal {
-               graalVersion '1.0.0-rc6'
-               mainClass "com.palantir.test.Main"
-               outputName "hello-world"
+               graalVersion '1.0.0-rc5'
```

Comment:
the tests in this class have side-effects in `~/.gradle`, so it's not clear you're really testing something here.

Reply:
(i.e., this test might pick up the rc6 thing from the test below, or an old, lingering binary, etc.)

Reply:
Yeah, not quite sure what a better test would look like -- can add some functionality like having a 'nativeImageVersion' task that emits the current version/can check that's what's in use. Also had a number of challenges getting both this test and the other to run.

Going to defer the work to #9 in the spirit of moving forward.